apps/web/src/app/(default)/check-mev-attack/page.tsx

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+export default function MEVCheckerAnalyzingPage() {
+  return (
+    <div className="flex items-center justify-between">
+      <div className="flex-1">
+        <h1 className="font-primary text-3xl leading-7 text-brand-30 mb-8">
+          Analyzing The Blocks
```

Comment:
can we map all the available messages and poses from Figma?

Reply:
Seems out of scope for this PR based on the figma (this is just the analyzing step)